share/cmake/modules/install/Installsse2neon.cmake

```diff
@@ -16,7 +16,7 @@ include(FetchContent)
 set(FETCHCONTENT_BASE_DIR "${CMAKE_BINARY_DIR}/ext/build/sse2neon")
 FetchContent_Declare(sse2neon
   GIT_REPOSITORY https://github.com/DLTcollab/sse2neon.git
-  GIT_TAG        v1.6.0
+  GIT_TAG        227cc413fb2d50b2a10073087be96b59d5364aea
```

Comment:
Any reason you couldn't pick an official version number here?

Reply:
Long story short, we uncovered some issues with 1.7.0 in Blender on various platforms, and needed to update to a non-versioned commit, as they only version sse2neon once a year - the issues may or may not affect OCIO, but this is at least a known working and tested version that Blender uses.